seeders/20191118200736-demo-facility.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  up: queryInterface => queryInterface.bulkInsert('facilities', [{
```

Comment:
I asked why use `facilities` as table name instead of `hotels`? `facilities` can mean a lot of things.

Reply:
In the PT they are using `facilities`  but this can be changed

Reply:
we are using `hotels` now